src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Interop/MethodOptions.ts

```diff
@@ -0,0 +1,28 @@
+﻿export interface MethodOptions {
+    type: TypeInstance;
+    method: MethodInstance;
+    async?: { resolveId: string, rejectId: string, functionName: string }
```

Comment:
It looks like `functionName` always takes the value `"invokeJavaScriptCallback"`. Until we have a use case for varying this value, could we simplify this by making `functionName` not something that can have different values specified?